app-server/datamodel/src/main/scala/com/azavea/rf/datamodel/v2/schema/schema.scala

```diff
@@ -2,12 +2,14 @@ package com.azavea.rf.datamodel.v2.schema
 // AUTO-GENERATED Slick data model
 /** Stand-alone Slick data model for immediate use */
 object tables extends {
-  val profile = slick.driver.PostgresDriver
+  val profile = com.azavea.rf.datamodel.driver.ExtendedPostgresDriver
```

Comment:
I don't think these should change -- the new driver shouldn't be necessary for these old migrations to compile (e.g. enums weren't necessary before this PR)